keps/sig-docs/20191020-doc-policies-for-third-party-content.md

```diff
@@ -0,0 +1,339 @@
+---
+title: doc-policies-for-third-party-content
+authors:
+  - "@aimeeu"
+  - "@jimangel"
+  - "@sftim"
+  - "@zacharysarah"
+owning-sig: sig-docs
+reviewers:
+  - "@jaredbhatti"
+  - "@kbarnard10"
+approvers:
+  - "@cblecker"
+  - "@derekwaynecarr"
+  - "@dims"
+editor: "@zacharysarah"
+creation-date: 2019-10-20
+last-updated: 2019-10-20
+status: provisional
+---
+
+# doc-policies-for-third-party-content
+
+## Table of Contents
+
+<!-- toc -->
+  - [Summary](#summary)
+  - [Introduction](#introduction)
+  - [Motivation](#motivation)
+    - [Goals](#goals)
+    - [Non-Goals](#non-goals)
+  - [Proposal](#proposal)
+    - [User Stories](#user-stories)
+      - [Story 1 (fictional)](#story-1-fictional)
+      - [Story 2 (fictional)](#story-2-fictional)
+      - [Story 3 (actual)](#story-3-actual)
+      - [Story 4 (actual)](#story-4-actual)
+      - [Story 5 (actual)](#story-5-actual)
+      - [Story 6 (actual)](#story-6-actual)
+    - [Implementation Details/Notes/Constraints](#implementation-detailsnotesconstraints)
+    - [Risks and Mitigations](#risks-and-mitigations)
+  - [Design Details](#design-details)
+    - [Graduation Criteria](#graduation-criteria)
+- [Drawbacks](#drawbacks)
+  - [Alternatives](#alternatives)
+<!-- /toc -->
+
+**Note:** This KEP does not target any release; SIG Docs follows a continuous
+release process for website content.
+
+## Summary
+
+This KEP seeks consensus on how Kubernetes docs handle two types of content:
+
+1. Content from or about third-party providers ("third-party content")
+
+Minimize and eliminate third-party content except when necessary for Kubernetes 
+to function in-project.
+
+2. Content hosted on multiple sites ("dual-sourced content")
+
+Minimize and eliminate dual-sourced content except when necessary for Kubernetes
+to function in-project.
+
+**Note:** This KEP defines "in project" to mean projects in the Kubernetes org, 
+which includes the [kubernetes](https://github.com/kubernetes) and 
+[kubernetes-sigs](https://github.com/kubernetes-sigs) repositories.
+
+## Introduction
+
+Kubernetes documentation teaches Kubernetes users about how
+Kubernetes works, how to use in-project Kubernetes features, and how to
+build on top of Kubernetes infrastucture.
+
+Feature docs are not a place for vendor pitches. Nonetheless, SIG Docs sometimes
+receives pull requests to place advertising-like content on the Kubernetes
+website. Some PRs clearly do not belong in feature docs, but other
+instances are less clear.
+
+Feature docs also contain dual-sourced content. A good practice for code
+project docs is to host single-sourced content only, and to provide
+links to other providers’ single-sourced content. This simplifies
+version management and reduces the work required to maintain content.
+
+This KEP proposes that Kubernetes documentation sets these boundaries:
+
+- Kubernetes documentation includes 
```

Comment:
I don't understand this line... is this a fragment or am I misreading it?

Reply:
Nope, it's a fragment. I've removed it.